static/schemas/2.0/Node.2.0.json

```diff
@@ -0,0 +1,35 @@
+{
+    "title": "Node.2.0",
+    "definitions": {
+        "Node": {
+            "description": "A node discovered by RackHD",
+            "type": "object",
+            "properties": {
+                "name": {
+                    "description": "Name of the node",
+                    "type": "string"
+                },
+                "type": {
+                    "description": "Type of node",
+                    "type": "string",
+                    "enum": ["compute", "switch", "dae", "pdu", "mgmt", "enclosure"]
```

Comment:
Actually,  I only added Redfish type support for OBM settings.   The below schema definition for OBM settings will require redfish as a type on the subsequent pass of the schemas.